docs/content/preview/drivers-orms/csharp/_index.md

```diff
@@ -19,6 +19,7 @@ The following projects can be used to implement C# applications using the Yugaby
 
 | Project | Documentation and Guides | Latest Driver Version | Supported YugabyteDB Version |
 | :------ | :----------------------- | :-------------------- | :--------------------------- |
+| YugabyteDB C# Driver for YSQL [Recommended] | [Documentation](ysql/) <br /> [Reference](../../reference/drivers/csharp/yb-npgsql-reference/) |
```

Comment:
@Sfurti-yb - Can I get the latest Driver version?

Reply:
Yes the latest version is `8.0.0-yb-1-beta`. For YugabyteDB version - `2.8 and above`